src/schema/global/typeDefs/index.ts

```diff
@@ -102,6 +102,22 @@ export const Enums = gql`
   }
 `;
 
+export const BannerPage = gql`
+  type BannerPage {
```

Comment:
In the other place most (if not all) of these fields were marked as deprecated? So when the new data comes and where?